src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1146,6 +1154,46 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/request-content-url':
+    post:
+      x-alfresco-since: "7.0"
+      tags:
+        - nodes
+      summary: Generate a direct access content url
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.0 and newer versions.
+
+        Generate a direct access content url for the given **nodeId** with the provided **expiryTime**.
+
+        **Note:** If an **expiryTime** isn't provided then a default of 28800 seconds is used.
+        **Note:** It is up to the actual ContentStore implementation if it can fulfil this request or not.
+      operationId: request-content-url
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/nodeDirectAccessExpiryTimeParam'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/NodeDirectAccessEntry'
+        '400':
+          description: |
+            Invalid parameter: **nodeId** is not a valid format or **expiryTime** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission for **nodeId**
+        '404':
+          description: |
+            **nodeId** does not exist
+        '501':
+          description: The actual ContentStore implementation can't fulfil this request.
```

Comment:
For consistency, the descriptions for the responses don't usually have a `.` at end